### PR TITLE
fix start-deployFTBA

### PR DESCRIPTION
**Pull request by IronMine**
fixes : #3123

### CODE REVIEW

**Comment by IronMine** [comment]
let me check just to be sure !

File: scripts/start-deployFTBA [hunk 1]
```diff
@@ -45,7 +45,7 @@ if isTrue "$FTB_FORCE_REINSTALL" || ! [ -f "${ftbInstallMarker}" ] || [ "$(cat "
     log "Downloading FTB installer"
     if [ "$(uname -m)" == "aarch64" ]; then
       log "Downloading ARM installer"
-      $arm=/arm
+      $arm="/arm"
```
Comment (itzg) [inline scripts/start-deployFTBA:48]: And remove that leading $ I think

(Some languages include that there so now I'm getting myself confused 😀)

Comment (IronMine) [reply scripts/start-deployFTBA]: yes because i was doing some php ealier this day :/

**Comment by IronMine** [comment]
really sorry for that i should have tested this ! But this makes me want to add a lot more tests ^^

**Comment by itzg** [comment]
> really sorry for that i should have tested this ! But this makes me want to add a lot more tests ^^
> 
> 

No problem at all. Take a look at the setuponly tests if you're interested...however, it's a balancing act since too many of them ran fairly slow.

**Review by itzg — APPROVED** [review]

**Pull request merged**